projects/core/src/store-finder/service/google-map-renderer.service.spec.ts

```diff
@@ -1,12 +1,12 @@
 import { GoogleMapRendererService } from './google-map-renderer.service';
 import { TestBed, fakeAsync, tick } from '@angular/core/testing';
 import { ExternalJsFileLoader } from './external-js-file-loader.service';
-import { Observable, of } from 'rxjs';
 import { StoreDataService } from '../facade/store-data.service';
-import { OccE2eConfigurationService } from '../occ/e2e/e2e-configuration-service';
+import { StoreFinderConfig } from '../config/store-finder-config';
+import { defaultStoreFinderConfig } from '../config/default-store-finder-config';
```

Comment:
maybe we could import this config this way: `import { defaultStoreFinderConfig as config } ...` - we use this value a few times and I think this approach has a better readability

Reply:
good point, done